launcher/src/test/java/org/apache/brooklyn/launcher/AbstractBrooklynLauncherRebindTest.java

```diff
@@ -184,7 +190,87 @@ protected File newTmpBundle(Map<String, byte[]> files, VersionedName bundleName,
         }
         return bf;
     }
+
+    BundleFile.Builder bundleBuilder() {
+        return new BundleFile.Builder(this);
+    }
     
+    protected static class BundleFile {
```

Comment:
nice.  maybe make its own class?

Reply:
It's currently tied to `AbstractBrooklynLauncherRebindTest`: it uses `test.newTmpBundle(...)`, which stores the created bundle file in the field `tmpFiles` so that they are deleted in `tearDown`.

Might well be worth extracting this into a utility that is re-used across multiple tests instead. I'll stick with it as-is for now, until we want to re-use it somewhere else, and we break that `tmpFiles` link.